cylc/flow/job_file.py

```diff
@@ -314,10 +313,8 @@ def _write_global_init_script(cls, handle, job_conf):
         """Global Init-script."""
         global_init_script = job_conf['platform']['global init-script']
         if cls._check_script_value(global_init_script):
-            handle.write("\n\ncylc__job__inst__global_init_script() {")
```

Comment:
The corresponding line which runs this function in `job.sh` needs to be removed

Reply:
Done.